src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.cpp

```diff
@@ -95,19 +95,20 @@ void ShenandoahScanRememberedTask::work(uint worker_id) {
   // This sets up a thread local reference to the worker_id which is necessary
   // the weak reference processor.
   ShenandoahParallelWorkerSession worker_session(worker_id);
+  ShenandoahWorkerTimingsTracker x(ShenandoahPhaseTimings::init_scan_rset, ShenandoahPhaseTimings::ScanClusters, worker_id);
 
   ShenandoahObjToScanQueue* q = _queue_set->queue(worker_id);
   ShenandoahObjToScanQueue* old = _old_queue_set == NULL ? NULL : _old_queue_set->queue(worker_id);
   ShenandoahMarkRefsClosure<YOUNG> cl(q, _rp, old);
-  RememberedScanner *rs = ShenandoahHeap::heap()->card_scan();
+  RememberedScanner *scanner = ShenandoahHeap::heap()->card_scan();
```

Comment:
While you are at it, can you put `*` to lean to the type? `RememberedScanner* scanner`.